src/rabbit_mgmt_util.erl

```diff
@@ -810,7 +810,13 @@ decode(<<"">>) ->
     {ok, #{}};
 decode(Body) ->
     try
-        {ok, rabbit_json:decode(Body)}
+        %% handle double encoded JSON, see rabbitmq/rabbitmq-management#839
+        case rabbit_json:decode(Body) of
+            Val when is_map(Val)    -> {ok, Val};
+            Val when is_list(Val)   -> {ok, maps:from_list(Val)};
```

Comment:
Maybe we should not assume that the list is a proplist… It is perfectly fine to pass an array as a JSON-encoded body. But we don't currently have any API endpoints that uses such a payload, so… @lukebakken